src/intTest/java/com/surrealdb/refactor/DemoScenarioTest.java

```diff
@@ -0,0 +1,63 @@
+package com.surrealdb.refactor;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import com.surrealdb.BaseIntegrationTest;
+import com.surrealdb.refactor.driver.BidirectionalSurrealDB;
+import com.surrealdb.refactor.driver.StatelessSurrealDB;
+import com.surrealdb.refactor.driver.SurrealDBFactory;
+import com.surrealdb.refactor.driver.UnauthenticatedSurrealDB;
+import com.surrealdb.refactor.types.Credentials;
+import com.surrealdb.refactor.types.Param;
+import com.surrealdb.refactor.types.Value;
+import org.junit.jupiter.api.Test;
+
+import java.net.URI;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DemoScenarioTest extends BaseIntegrationTest {
+    @Test
+    public void testDemoScenario() throws Exception {
+        // Setup
+        URI address = getHttp().orElseThrow(() -> new IllegalStateException("No HTTP server configured"));
+        UnauthenticatedSurrealDB<BidirectionalSurrealDB> unauthenticated = new SurrealDBFactory().connectBidirectional(address);
+
+        // Authenticate
+        BidirectionalSurrealDB surrealDB = unauthenticated.authenticate(new Credentials("admin", "admin"));
+
+        // Create a multi-statement query
+        StringBuilder query = new StringBuilder("INSERT person:lamport VALUES {'name': 'leslie'};\n");
```

Comment:
The new line number isn't necessary FYI. Semicolons are. We can always have our own statement builder, which could be type enforced and result in correct strings.